core/aws_ddk_core/stages/sqs_lambda.py

```diff
@@ -53,6 +53,7 @@ def __init__(
         message_group_id: Optional[str] = None,
         lambda_function_errors_alarm_threshold: Optional[int] = 5,
         lambda_function_errors_alarm_evaluation_periods: Optional[int] = 1,
+        lambda_function_errors_alarm_enabled: Optional[bool] = True,
         function_props: Optional[Dict[str, Any]] = {},
```

Comment:
Shouldn't this constructor also have the `alarms_enabled` parameter, which will then be passed to `super`?

Reply:
I'm playing around with just inheriting `alarms_enabled` from `DataStage`, put this back in `Draft`

Reply:
Similar to my question above, but do you think we should expose `alarms_enabled` in all stages or use `kwargs`?

Reply:
Personally, I'd rather expose it explicitly than use `kwargs`. Standard CDK constructs have parameters explicitly defined in all subclass constructors. It's a lot easier for them though, as they write these constructs in Typescript, where the parameters are inherited between classes, and then translated it to Python using JSII. 

However, I think it's still worth it in order to have very explicit signatures. If a customer just wants to view the docs to see the parameters for `SQSLambdaStage`, we want them to see `alarms_enabled` as a parameter, not just `kwargs`.